.github/workflows/check-merge-freeze.yml

```
@@ -0,0 +1,36 @@
+name: Merge freeze
+
+on:
+  pull_request:
+    types:
+      [
+        opened,
+        ready_for_review,
+        synchronize,
+        reopened,
+        labeled,
+        unlabeled,
+        enqueued,
+      ]
```

you could just use a normal yaml list instead
```suggestion
    types:
      - opened
      - ready_for_review
      - synchronize
      - reopened
      - labeled
      - unlabeled
      - enqueued
```